src/contracts/AccessList.ts

```diff
@@ -41,6 +39,33 @@ export class AccessListContract extends SmartContract {
     return await accessListContract.tokenURI()
   }
 
+  /**
+   * Get Owner
+   * @return {Promise<string>} Owner
+   */
+  public async getOwner(accessListAddress: string): Promise<string> {
+    const accessListContract = this.getContract(accessListAddress)
+    return await accessListContract.owner()
+  }
+
+  /**
+   * Get Name of Access list
+   * @return {Promise<string>} Name
+   */
+  public async getName(accessListAddress: string): Promise<string> {
+    const accessListContract = this.getContract(accessListAddress)
+    return await accessListContract.name()
+  }
+
+  /**
+   * Get Symbol of Access list
+   * @return {Promise<string>} Symbol
+   */
+  public async getSymbol(accessListAddress: string): Promise<string> {
+    const accessListContract = this.getContract(accessListAddress)
+    return await accessListContract.symbol()
+  }
+
   /**
    * Mint ERC721 contract
```

Comment:
mint new token, or add address to accesslist

it's not minting another erc721 contract

Reply:
I'll modify